src/locale/ar.json

```diff
@@ -0,0 +1,318 @@
+  "loading_routes": "تحميل المسارات",
+  "loading_stops": "تحميل محطات التوقف",
+  "loading_gaps": "تحميل فجوات الرحلات",
+  "timestamp_list": "قائمة بجميع أوقات التوقف في المحطة المحددة",
+  "timestamp_target": "⌚ وقت البحث",
+  "timestamp_gtfs": "🕛 وقت التوقف المجدول",
+  "timestamp_siri": "🚌 وقت التوقف الفعلي",
+  "loading_hits": "تحميل أوقات الرحلات",
+  "line_not_found": "لم يتم العثور على الخط",
+  "hits_not_found": "لم يتم العثور على رحلات مجدولة أو فعلية",
+  "planned_time": "الوقت المجدول",
+  "planned_status": "الحالة",
+  "ride_as_planned": "✔️ خرجت كما هو مخطط لها",
+  "ride_missing": "❌ رحلة مفقودة",
+  "ride_in_future": "🕛 رحلة في المستقبل",
+  "ride_extra": "🧐 رحلة غير مجدولة",
+  "ride_duped": "❇️ رحلة مزدوجة",
+  "checkbox_only_gaps": "الفجوات فقط",
+  "dashboard_tooltip_content": "يحتوي كل خط في إسرائيل على نظام تحديد المواقع العالمي (GPS) الذي يبلغ عن موقع الحافلة كل بضع لحظات.\nإذن ما هي الرحلة الفائتة؟ إنها رحلة تم التخطيط لها ولكن لم يتم تسجيلها في بيانات GPS. يمكنك رؤيتها في التطبيق، على سبيل المثال، ولكن عندما تنتظر في المحطة، لن تصل أبدًا",
+  "worst_lines_page_title": "الخطوط الأقل كفاءة",
+  "rides_planned": "الرحلات المجدولة",
+  "rides_history": "تاريخ الرحلات",
+  "rides_actual": "الرحلات التي خرجت",
+  "rides_missing": "الرحلات المفقودة",
+  "dashboard_page_graph_title": "نسبة خروج إجمالي الرحلات حسب الوقت",
+  "from_date": "من تاريخ",
+  "to_date": "إلى",
+  "watch_locations_in_range": "عرض مواقع الحافلات في النطاق",
+  "minutes": "دقائق",
+  "minutes_5": "5 دقائق",
+  "show_x_bus_locations": "مواقع الحافلات",
+  "from_time_x_to_time_y": "من XXX إلى YYY",
+  "choose_start_time": "اختر وقت البدء",
+  "loading_times_tooltip_content": "تحميل أوقات الحافلات. الأوقات الحالية متاحة للاستخدام",
+  "group_by_hour_tooltip_content": "تجميع حسب الساعة",
+  "start": "بداية",
+  "end": "نهاية",
+  "about_title": "حول",
+  "public_appeal_title": "نداء عام",
+  "donate_title": "للتبرعات",
+  "report_a_bug_title": "الإبلاغ عن خطأ",
+  "how_to_donate_title": "كيف يمكنني التبرع؟",
+  "how_to_donate_text": "وراء الكواليس، يوجد فريق صغير ومتفانٍ يهتم بتنظيم لقاءات التطوير والهاكاثونات وصيانة الخوادم، ويضع قدميه بين المكاتب الحكومية لإقناع والضغط والتأثير لإصدار المزيد والمزيد من المستودعات؛ والتأكد من أن وسائل الإعلام تعرف عملنا وتعرف أيضًا كيفية الاتصال بنا للحصول على البيانات والتحليلات، وتقوم بالكثير من العمل المستمر حتى تحدث مشاريعنا فرقًا في العالم. نحن بحاجة إلى دعم المجتمع لمواصلة العمل من أجل حكومة أكثر انفتاحًا. كن فارسًا للشفافية وانضم إلى الدعم الشهري.",
+  "donate": "تبرع",
+  "donation_through_bank_title": "التبرع عن طريق التحويل المصرفي",
+  "donation_through_bank_reccomendation": "يوصى بالتبرع عن طريق التحويل المصرفي إلى الحساب المصرفي لورشة المعرفة العامة، حيث أن هذه العمولة أقل",
+  "donate_to_hasadna": "تبرع لـ Hasadna",
+  "donation_link": "رابط التبرع",
+  "bank": "البنك",
+  "branch": "الفرع",
+  "account": "رقم الحساب",
+  "account_name": "اسم الحساب",
+  "donation_through_bank_details_bank": "بنك يو (26)",
```

Comment:
I'm not sure there's such an option when selecting a bank in Israel, we can add both names in Hebrew and Arabic together although the number makes it pretty easy regardless of the translation.

Reply:
We should still add some clarification or keep the Hebrew values in addition because it's not necessarily translated in all platforms, same goes for Rehavia, in all languages.